test_files/expected/simplify_xpaths_success_3.txt

```diff
@@ -1,31 +1,31 @@
-rings:occultation_type
-rings:occultation_direction
-rings:time_series_direction
-rings:planetary_occultation_flag
-rings:data_quality_score
-rings:ring_plane
-rings:star_name
-rings:fresnel_scale
-rings:projected_star_diameter
-rings:sigma_projected_star_diameter
-rings:fractional_error_star_counts
-rings:time_constant_type
-rings:time_constant
-rings:sigma_time_constant
-rings:minimum_wavelength
-rings:maximum_wavelength
-rings:reference_time_utc
-rings:minimum_observed_event_time
-rings:maximum_observed_event_time
-rings:observed_event_start_tdb
-rings:observed_event_stop_tdb
-rings:earth_received_start_time_utc
-rings:earth_received_stop_time_utc
-rings:minimum_ring_radius
-rings:maximum_ring_radius
-rings:minimum_ring_longitude
-rings:maximum_ring_longitude
-rings:minimum_observed_ring_azimuth
-rings:maximum_observed_ring_azimuth
-rings:observed_ring_elevation
-rings:light_source_incidence_angle
+rings:occultation_type<1>
```

Comment:
Same here - shouldn't these be simplified?

Reply:
Discussed in Tuesday's meeting.